UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -213,5 +226,86 @@ public async Task WhenGetBatchInfoBasedOnProducts_ThenReturnsSearchBatchResponse
         }
 
         #endregion GetBatchInfoBasedOnProducts
+
+        #region SearchReadMeFilePath
+        [Test]
+        public async Task WhenSearchReadMeFilePath_ThenSearchReadMeFilePathReturnsEmptyStringResponse()
```

Comment:
Kindly change WhenSearchReadMeFilePath_ThenSearchReadMeFilePathReturnsEmptyStringResponse to
WhenInvalidRequest_ThenSearchReadMeFileReturnsEmptyFilePath()

Reply:
Done